.github/workflows/docs-pr.yml

```diff
@@ -39,6 +39,36 @@ jobs:
           # Update hand-written documentation
           rm -rf docs/docs/get-details/conduit/
           cp -r conduit/docs docs/docs/get-details/conduit
+
+          # Copy the plugin docs using this ChatGPT special.
+          # Set the directory to search
+          search_dir="conduit/plugins"
+          target_base_dir=docs/docs/get-details/conduit/plugins
```

Comment:
is this copying READMEs from `conduit/plugins` to `docs`? what is this path `docs/docs/get-details/conduit/plugins`?

Reply:
It's in the docs repo: https://github.com/algorand/docs/tree/staging/docs/get-details/conduit/plugins